src/processes.md

```diff
@@ -0,0 +1,14 @@
+# Questions about your development processes
+
+## How should I use tools differently from C++?
+
+* *Use `rustfmt` automatically everywhere.* While in C++ there are many
+  different coding styles, the Rust community is in agreement (at least,
+  they're in agreement that it's a good idea to be in agreement). That
+  is codified in `rustfmt`. Use it, automatically, on every submission.
+* *Use `clippy` somewhere*. Its lints are useful.
```

Comment:
I like to create a shell alias, or even a target in a Makefile (I have no shame!), that does `cargo build && cargo test && cargo clippy` in 1 command. I.e. I never don't use them!

Reply:
I have github triggers to prevent me landing code unless clippy is clean. It's nice.